pages/index.html

```diff
@@ -4,9 +4,25 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Ada Barries</title>
+  <link href="/styles/index_style.css" rel="stylesheet" />
 </head>
-<body>
-  
+<body class="container">
+  <main class="splash-page">
+    <section class="splash-image">
+      <img src="/images/timepixel.png" alt="pixel image of the Time Aspect symbol">
+    </section>
```

Comment:
Style: This one also ought to be a `div` (or styles directly applied to the image).